datasets/management/commands/searchvector.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.postgres.search import SearchVector
+from django.core.management.base import BaseCommand
+
+from datasets.models import Gazette
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
```

Comment:
Quando puder, adiciona um comentário aqui explicando como funciona esse comando. Vamos rodá-lo todos os dias? A cada uma hora? Precisa ser executado para todos os objetos sempre ou podemos deixar de fora os que já tem índices?

Reply:
Tem uma forma de escrever uma migração mais manual, e fazer isso de forma automática toda vez que atualizar o documento. Mas, tenho minhas dúvidas sobre isso ser realmente a melhor opção no nosso caso. Estou fazendo alguns testes de desempenho.

Sobre rodar o comando, eu gostei dessa ideia. Peguei a referência lá no jarbas :)
Cada vez que rodamos esse comando ele recria os índices. Se a ideia de fazer de vez com trigger não se mostrar muito legal, pensei em adotar uma estratégia com timestamp e reprocessar apenas os que não estão no índice ou que foram atualizados recentemente. A frequẽncia vai depender da nossa disponibilidade de recursos e documentos. Ele processa os 8k atuais em menos de 3 minutos. E só fica disponível no indice depois desse processamento. Penso que, se for realmente para usar o comando, que podemos combinar com a ideia de reprocessar os dados novos apenas assim que o crawler terminar.

Reply:
É, usar triggers seria massa pq não precisaríamos nos preocupar com um outrto comando mas não faço ideia da performance (na real, a gente não vai ter mais que 50 mil registros entrando no banco por dia, então nem sei se precisamos escovar os bits assim hehe). Sobre os triggers, a gente tem como opções o _signals_ do Django e também o `finished_spider`, pra quando as coletas terminam.

Reply:
Estou mais inclinado a usar os signals do crawler mesmo. Por que dá pra fazer a operação em lote. Mais rápido que ficar atualizando de um por um.

Reply:
Como não vamos ter tantos registros novos por dia, optei por começar com o trigger.
Caso vire um gargalo para o armazenamento podemos revisitar isso mais pra frente.

Mantive o comando como um backup. Em produção, por exemplo, que já tem dados, vamos precisar rodar ele para gerar o índice. Pode ser útil em futuras migrações.